app/admin/widgets/Filter.php

```diff
@@ -416,6 +437,27 @@ public function applyScopeToQuery($scope, $query)
                 }
 
                 break;
+                
+            case 'daterange':
+                $value = explode('|', $scope->value);
```

Comment:
Can you use ` - ` separator instead to match the same format expected by the daterangepicker.js plugin.